stripe-connect/src/main/java/com/stripe/android/connectsdk/EmbeddedComponentManager.kt

```diff
@@ -0,0 +1,4 @@
+package com.stripe.android.connectsdk
+
+@PrivateBetaConnectSDK
+class EmbeddedComponentManager internal constructor()
```

Comment:
Generally, we annotate these with @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) before they're released. That will ensure that these are not actually included in our public API yet. Then, whenever you're ready to ship private beta, you can send out a PR which just removes the annotation and adds a changelog entry about the new feature/product being available. I'd recommend that approach here

Reply:
Great idea! I like this approach a lot

Reply:
agreed!